src/stories/components/events/NewEvent.tsx

```diff
@@ -16,10 +16,12 @@ interface NewEventProps {
 function checkDateValid(startTime?: Date, endTime?: Date) {
 	let fullDateText = "Not Set";
 	if (startTime && endTime) {
-		const dateText = dayjs(startTime).format("DD/MM/YYYY");
-		const startTimeText = dayjs(startTime).format("HH:mm");
+		const startTimeText =
+			dayjs(startTime).format("DD/MM/YYYY") +
+			" " +
+			dayjs(startTime).format("HH:mm");
 		const endTimeText = dayjs(endTime).format("HH:mm");
-		fullDateText = dateText + " " + startTimeText + "-" + endTimeText;
+		fullDateText = startTimeText + "-" + endTimeText;
```

Comment:
Check javascript literals
```suggestion
		fullDateText = `${startTimeText} - ${endTimeText}`;
```